src/components/CurrencyInputPanel/FiatValue.tsx

```diff
@@ -29,17 +39,33 @@ export function FiatValue({
   const p = Number(fiatValue?.toFixed())
   const visibleDecimalPlaces = p < 1.05 ? 4 : 2
 
+  useEffect(() => {
+    if (isLoading) {
+      const timeout = setTimeout(() => setShowLoadingPlaceholder(true), 200)
```

Comment:
possible this timeout will trigger a setState on an unmounted component--should do this https://stackoverflow.com/a/60907638

here's something similar I wrote today:
```
  useEffect(() => {
    let stale = false
    let timeoutId = 0
    if (animateBuyCryptoButtonBorder) {
      timeoutId = setTimeout(() => {
        if (stale) return
        acknowledge({ system: true })
      }, ms`2 seconds`) as unknown as number
      // as unknown as number is necessary so it's not incorrectly typed as a NodeJS.Timeout
    }
    return () => {
      stale = true
      clearTimeout(timeoutId)
    }
  }, [acknowledge, animateBuyCryptoButtonBorder])
```

Reply:
smart fix!! i was wondering about this....thank you thank you. done